vpr/src/route/multi_queue_priority_queue.cpp

```diff
@@ -19,23 +19,29 @@ void MultiQueuePriorityQueue::init_heap(const DeviceGrid& grid) {
     // TODO: Reserve storage for MQ_IO
 }
 
-bool MultiQueuePriorityQueue::try_pop(pq_node_t &pq_top) {
+bool MultiQueuePriorityQueue::try_pop(pq_prio_t &prio, RRNodeId &node) {
     auto tmp = pq_->tryPop();
     if (!tmp) {
         return false;
     } else {
-        pq_top = std::get<1>(tmp.get());
+        pq_index_t node_id;
+        std::tie(prio, node_id) = tmp.get();
+        node = RRNodeId(node_id);
         return true;
     }
 }
 
-void MultiQueuePriorityQueue::add_to_heap(const pq_node_t& hptr) {
-    pq_->push({hptr.cost, hptr});
+inline pq_index_t cast_RRNodeId_to_pq_index_t(RRNodeId node) {
+    return static_cast<pq_index_t>(std::size_t(node));
```

Comment:
Same comment as above. Please add a static assert at any point in which you assume that RRNodeId is 32 bits.